ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -109,12 +109,15 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
         I18n.t("serviceDetail.onUpdateEnabledChannelsFailure"),
```

Comment:
I found a bug on the first **if** statement.
Could you replace it with this?
```
// if an error incoming show toast and restore the previous values
    if (pot.isError(nextProps.profile) && !pot.isError(this.props.profile)) {
```
and check it works properly?